extensions/amp-render/1.0/storybook/Basic.amp.js

```diff
@@ -107,3 +107,48 @@ export const WithBindableSrc = () => {
 WithBindableSrc.story = {
   name: 'With bindable src',
 };
+
+export const WithAmpScriptSrc = () => {
+  return (
+    <>
+      <script
+        async
+        custom-element="amp-script"
+        src="https://cdn.ampproject.org/v0/amp-script-0.1.js"
+      ></script>
+      <meta
+        name="amp-script-src"
+        content="sha384-W71daLvaG1fVtSoGC4xq80vkUeKAZPo2hPEj4ofjtSOSfK1m02U0gQXp1ChLV1O8"
+      ></meta>
```

Comment:
I would prefer if we didn't hardcode the sha, bc of the roundtrip to edit the inline script below. Please do either of the following and let me know if it doesn't work (non-blocker):

1. **Generate sha on the fly**. Try using [`@ampproject/toolbox-script-csp`](https://www.npmjs.com/package/@ampproject/toolbox-script-csp). If that doesn't work on the client, let me know.

2. **Serve the script as a Storybook [static asset](https://storybook.js.org/docs/react/configure/images-and-assets#serving-static-files-via-storybook)**, since same-origin scripts loaded by src don't require the sha. (You may configure the static directory in our [Storybook runner task.)](https://github.com/ampproject/amphtml/blob/2298e0664ab937cb9f04971b6a4741b5cd2b2491/build-system/tasks/storybook/index.js)

Reply:
I've added a todo to try these out.

Reply:
Actually, disregard this. Generating the hash should be part of the part of the Storybook AMP addon: https://github.com/ampproject/storybook-addon-amp/pull/22

You can leave it as-is, but please add this TODO:

```suggestion
      {/* TODO(alanorozco): Remove hash once storybook-addon-amp#22 is mainlined */}
      <meta
        name="amp-script-src"
        content="sha384-W71daLvaG1fVtSoGC4xq80vkUeKAZPo2hPEj4ofjtSOSfK1m02U0gQXp1ChLV1O8"
      ></meta>
```

Reply:
Done